src/components/slider/sliderComponent.js

```diff
@@ -0,0 +1,47 @@
+import React ,{useState} from 'react';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faChevronLeft } from '@fortawesome/free-solid-svg-icons';
+import { faChevronRight } from '@fortawesome/free-solid-svg-icons';
+import { faCircle } from '@fortawesome/free-solid-svg-icons';
+import Slide from './slideComponent.js';
+import '../../assets/slide.css';
+
+const Slider=(props)=>{ 
```

Comment:
I think it is best if you destructure your props here IE:
```suggestion
const Slider=( { elements } )=>{ 
```
and speaking about the prop names as this is a generic component I believe changing the name to one that is more generic would be the best, can be something like data, elements